Workbooks/AKS/Syslog/Syslog.workbook

```diff
@@ -211,13 +216,19 @@
       "type": 3,
       "content": {
         "version": "KqlItem/1.0",
-        "query": "Syslog\r\n| where HostName in ({HostName}) or '*' in ({HostName})\r\n| where Facility in ({Facility}) or '*' in ({Facility})\r\n| where SeverityLevel in ({SeverityLevel}) or '*' in ({SeverityLevel})\r\n| summarize count(SeverityLevel) by SeverityLevel \r\n| extend jkey = 1\r\n| join (Syslog\r\n| where HostName in ({HostName}) or '*' in ({HostName})\r\n| where Facility in ({Facility}) or '*' in ({Facility})\r\n| where SeverityLevel in ({SeverityLevel}) or '*' in ({SeverityLevel})\r\n  | make-series Trend = count(SeverityLevel) default = 0 on TimeGenerated from {TimeRange:start} to {TimeRange:end} step {TimeRange:grain} by SeverityLevel) on SeverityLevel\r\n  //| project Emergency=emerg, Alert=alert, Critical=crit, Error=err, Warning=err, Warning=warning, Notice=notice, Informational=info, Debug=debug",
+        "query": "Syslog\r\n| where HostName in ({HostName}) or '*' in ({HostName})\r\n| where Facility in ({Facility}) or '*' in ({Facility})\r\n| where SeverityLevel in ({SeverityLevel}) or '*' in ({SeverityLevel})\r\n| summarize count(SeverityLevel) by SeverityLevel \r\n| extend jkey = 1\r\n| join (Syslog\r\n| where HostName in ({HostName}) or '*' in ({HostName})\r\n| where Facility in ({Facility}) or '*' in ({Facility})\r\n| where SeverityLevel in ({SeverityLevel}) or '*' in ({SeverityLevel})\r\n  | make-series Trend = count(SeverityLevel) default = 0 on TimeGenerated from {timeRange:start} to {timeRange:end} step {timeRange:grain} by SeverityLevel) on SeverityLevel\r\n  //| project Emergency=emerg, Alert=alert, Critical=crit, Error=err, Warning=err, Warning=warning, Notice=notice, Informational=info, Debug=debug",
         "size": 4,
+        "title": "Summary by Severity Level",
         "noDataMessage": "No results returned - Enable Syslog or extend time-range",
         "noDataMessageStyle": 5,
-        "timeContextFromParameter": "TimeRange",
-        "exportFieldName": "SeverityLevel1",
-        "exportParameterName": "SeverityLevel",
+        "timeContextFromParameter": "timeRange",
+        "exportedParameters": [

```

Comment:
Can we change this part to this instead so it will not break the queries:

"exportFieldName": "Severity Level",        
"exportParameterName": "SeverityLevel",        
"exportDefaultValue": "\"*\"",